Superlib.js

```diff
@@ -0,0 +1,633 @@
+				itemInfo[url] = {
+					pdfurl: download.href
+				};
+			}
+		}
+		if (title.startsWith('《') && title.endsWith('》')) {
+			title = title.replace(/《|》/g, '');
+		}
+		
+		items[url] = title;
+	}
+	return found ? items : false;
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		var itemInfo = {};
+		Zotero.selectItems(getSearchResults(doc, false, itemInfo), function (items) {
+			if (items) {
+				if (items['']) {
+					delete items[''];
+				}
+				scrapeItem(Object.keys(items), itemInfo);
+			}
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function scrapeItem(items, itemInfo) {
+	ZU.processDocuments(items, function (doc, url) {
+		scrape(doc, url, itemInfo[url] ? itemInfo[url].pdfurl : null);
+	});
+}
+
+function scrape(doc, url, pdfurl) {
+	if (!url || url.length <= 0) {
+		return;
+	}
+	
+	var itemType = detectType(doc);
+	var item = new Zotero.Item(itemType);
+
+	item.url = url;
+	switch (sourceType(doc)) {
+		case '图书':
+			item.title = text(doc, '.tutilte');
+			var infos = text(doc, '.tubox dl');
+			infos = trim(infos);
+			for (var section0 of Object.values(infos.split('\n'))) {
+				if (!section0 || section0.trim().length <= 0) continue;
+		
+				let index = section0.indexOf('】');
+				if (index <= -1) continue;
+		
+				let key = section0.substr(0, index + 1).trim();
+				let value = section0.substr(index + 1).trim();
+				// Z.debug(key + ':' + value);
+				switch (key) {
+					case "【作　者】":
+						doPerson(item, value);
+						break;
+					case "【出版项】":
+						var vals = Object.values(value.split(/：|,/g));
+						item.publisher = vals[1];
+						item.date = vals[2];
+						break;
+					case "【ISBN号】":
+						item.ISBN = value;
+						break;
+					case "【形态项】":
+						item.numPages = value;
+						break;
+					case "【中图法分类号】":
+						item.archiveLocation = value.replace(/\(.*\)/, '');
+						break;
+						
+					default:
+						break;
+				}
+			}
+		
+			// 摘要
+			item.abstractNote = text(doc, '.tu_content').replace(/内容提要:\n*/g, '');
+			break;
+		case '期刊':
+			item.title = ZU.trimInternal(text(doc, 'h1.title'));
+			infos = doc.querySelectorAll('#m_top li');
+			for (var section1 of infos) {
+				let content = ZU.trimInternal(section1.textContent);
+				if (!content || content.length <= 0) continue;
+		
+				let index = content.indexOf('】');
+				if (index <= -1) continue;
+		
+				content = trim(content);
+				let key = content.substr(0, index + 1).trim();
+				let value = content.substr(index + 1).trim();
+				switch (key) {
+					case "【作 者】":
+						doPerson(item, value);
+						break;
+					case "【刊 名】":
+						item.publicationTitle = value;
+						break;
+					case "【期 号】":
+						item.issue = value;
+						break;
+					case "【出版日期】":
+						item.date = value;
+						break;
+					case "【摘 要】":
+						item.abstractNote = value;
+						break;
+					case "【关键词】":
+						doTag(item, value);
+						break;
+					case "【影响因子】":
+						item.extra = '影响因子: ' + value;
```

Comment:
What gets stored in this field? The unfortunate fact of Extra is that keys can't be localized, but I'll leave it up to others - @dstillman? - whether the key should be in English or Chinese.

Reply:
Generally we'd want English, since some keys in Extra will be converted to real Zotero values that get added later, and some are even already automatically used in citations (e.g., "Original Date:"). But if we think there's no chance of that for a given key, and we think the audience is going to be entirely Chinese-speaking, Chinese is fine. The translator is already saving all other values in Chinese.